components/onboarding/BasicInformations.tsx

```diff
@@ -0,0 +1,120 @@
+'use client';
+
+import { EOnboardingStep } from '@/types/onboarding-step';
+import {
+  CldUploadButton,
+  CloudinaryUploadWidgetInfo,
+  CloudinaryUploadWidgetResults,
+} from 'next-cloudinary';
+import Image from 'next/image';
+import { useState } from 'react';
+import { useFormContext } from 'react-hook-form';
+import RHFInput from '../RHFInputs/RHFInput';
+import { Button } from '../ui/button';
+import { Label } from '../ui/label';
+import { IUserOnboarding } from '@/lib/zod/onboarding-schema';
+
+// ----------------------------------------------------------------
+
+interface IBasicInformationsProps {
+  handleChangeStep: (
+    data: Partial<IUserOnboarding>,
+    newStep: EOnboardingStep
+  ) => void;
+}
+
+const BasicInformations: React.FC<IBasicInformationsProps> = ({
+  handleChangeStep,
+}) => {
+  const { LEARNING_GOALS } = EOnboardingStep;
+  const { trigger, setValue, getValues } = useFormContext();
+  const [uploadedImage, setUploadedImage] = useState('');
+
+  const onSuccessUpload = (
+    result: CloudinaryUploadWidgetResults,
+    options: any
+  ) => {
+    console.log('result', result);
+    console.log('options', options);
+
+    if (!result || !result?.info) throw new Error('Image not uploaded!');
```

Comment:
```suggestion
    if (!result?.info) throw new Error('Image not uploaded!');
```
This should do the same thing